pkg/util/shell/shell.go

```diff
@@ -90,7 +90,7 @@ func GetPrefixSuffixDelimiterForSet(userShell string, pathVar bool) (prefix, suf
 		prefix = "SET "
 		suffix = "\n"
 		if pathVar {
-			suffix = ";%PATH%" + suffix
+			suffix = ";%PATH%;" + suffix
```

Comment:
I don't think the ending `;` is needed. Doing this creates an empty token at the end of the path as it does `;;`